theano/gpuarray/tests/test_pickle.py

```diff
@@ -29,7 +29,7 @@
         cuda_ndarray.use('gpu', default_to_move_computation_to_gpu=False,
                          move_shared_float32_to_gpu=False,
                          enable_cuda=False)
-        theano.sandbox.gpuarray.init_dev('cuda')
```

Comment:
Can you make a PR to Simon PR and fix this?

On Fri, May 6, 2016 at 3:46 PM, abergeron <notifications@github.com> wrote:

> In theano/gpuarray/tests/test_pickle.py
> <https://github.com/Theano/Theano/pull/4454#discussion_r62381295>:
>
> > @@ -29,7 +29,7 @@
> >          cuda_ndarray.use('gpu', default_to_move_computation_to_gpu=False,
> >                           move_shared_float32_to_gpu=False,
> >                           enable_cuda=False)
> > -        theano.sandbox.gpuarray.init_dev('cuda')
>
> I'm ok with that.
>
> Though I suspect that this code is why we saw the testsuite use two GPUs
> for no reason.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/4454/files/a5b12a87e8389f7d78a7b51d541ae840854b3fc1#r62381295>
>